sdk/node/Libplanet.Node/Options/GenesisOptions.cs

```diff
@@ -26,6 +27,12 @@ public sealed class GenesisOptions : OptionsBase<GenesisOptions>
 
     [Description(
         $"The path of the genesis block, which can be a file path or a URI." +
-        $"This property cannot be used with {nameof(GenesisKey)}.")]
+        $"This property cannot be used with {nameof(GenesisKey)} and " +
+        $"{nameof(GenesisConfigurationPath)}.")]
     public string GenesisBlockPath { get; set; } = string.Empty;
+
+    [Description(
+        $"The path of the genesis configuration, which can be a file path or a URI." +
+        $"This property cannot be used with {nameof(GenesisKey)} and {nameof(GenesisBlockPath)}.")]
+    public string GenesisConfigurationPath { get; set; } = string.Empty;
```

Comment:
If there are many ways to create Genesis blocks, it may be better to create a type variable rather than increase the properties.
If the properties increase, the complexity will increase because the exception handling code must be implemented.
For example,
```csharp
enum GenesisBlockType
{
    Binary,
    BinaryUrl,
    Json,
    JsonUrl
}
```
I don't know what the best way is, but I think it's worth considering.

Reply:
I guess this suggestion should be dealt in one separated PR